src/PostList.js

```diff
@@ -0,0 +1,72 @@
+import { push } from "./util/router.js"
+import DeleteButton from "./button/DeleteButton.js"
+
+export default function PostList ({ $target, initialState}) {
+    const $postList = document.createElement('div')
+    $target.appendChild($postList)
+
+    this.state = initialState
+
+    this.setState = nextState => {
+        this.state = nextState
+        this.render()
+    }
+
+
+
+    this.render = () => {
+        $postList.innerHTML =`
+        <ul style ='
+        list-style: none; border: 1px solid black; border-radius: 2px;
+        margin: 0px 0px; padding: 0px 0px 0px 0px;
+                  ;
+        '>
+            ${this.state.map(post => `
+                <li style =' 
+                display: flex;
+                overflow: hidden;
+                text-overflow: ellipsis;
+                white-space: nowrap;
+                padding: 7px; cursor : pointer; user-select: none;
+                '
+                onmouseover="this.style.backgroundColor='#D8D8D8'"
+                onmouseout="this.style.backgroundColor='white'"
+                
+                data-id ="${post.id}">&#127817${post.title}&nbsp&nbsp
```

Comment:
```suggestion
                data-id ="${post.id}">&#127817 ${post.title}&nbsp&nbsp
```
제목 앞부분이 숫자인 경우 숫자가 출력되지 않습니다. 🍉 이모지와 함께 나오기 위해서 한 칸 띄워줘야 합니다!